integration/cluster.go

```diff
@@ -711,6 +711,15 @@ func (m *member) StopWithAutoLeaderTransfer(t *testing.T) {
 	plog.Printf("stopped %s (%s)", m.Name, m.grpcAddr)
 }
 
+// checkLeaderTransition waits for leader transition, returning the new leader ID.
+func checkLeaderTransition(t *testing.T, m *member, oldLead uint64) uint64 {
+	interval := time.Duration(m.s.Cfg.TickMs) * time.Millisecond
+	for m.s.Lead() == 0 || (m.s.Lead() == oldLead) {
```

Comment:
so this doesn't distinguish between a regular election and a transfer? Would that mean this test would pass even if the leader was hard stopped?